client/src/components/Reviews/Form.jsx

```diff
@@ -1,90 +1,146 @@
 import React, { useState, useEffect } from 'react';
-import { FormView } from '../styles/reviewstyles/FormView.styled.js';
-import { Background } from '../styles/reviewstyles/FormBackground.styled.js';
-import { SummaryBox } from '../styles/reviewstyles/SummaryBox.styled.js';
+import { ThumbNail, FullSize } from '../styles/reviewstyles/imageStyles.styled.js';
+import { FormModal, FormModalBackground, Star, FormTitle, FormGrid, FormRow, FormCol } from '../styles/reviewstyles/formStyles.styled.js';
+import axios from 'axios';
 
-const Form = ({ closeForm, form }) => {
+import StyleRating from './StyleRating.jsx';
+import StarRating from './StarRating.jsx';
+
+const Form = ({ closeForm, form, metaReview, currentProductId, }) => {
+  //form values
   const [summary, setSummary] = useState('');
   const [name, setName] = useState('');
   const [review, setReview] = useState('');
   const [email, setEmail] = useState('');
-  const [rating, setRating] = useState(null);
-  const [recommend, setrecommend] = useState(false);
-
-  const [maxFiles, setFiles] = useState(0);
+  const [recommend, setRecommend] = useState(null);
+  const [photoArray, setPhotoArray] = useState('');
+  //star rating values
+  const [starRating, setStarRating] = useState(0);
+  //style rating values
+  const [charRating, setCharRating] = useState({});
+  //form validator hook
+  const [formValid, setFormValid] = useState(false);
 
+  //universal input handler
   let handleChange = (event, setState) => {
-    setState(prevState => event.target.value);
-    console.log('review', review, 'summary', summary, 'name', name);
+    setState(event.target.value);
   };
 
-  let handleRating = () => {
-    console.log('');
+  //validates that the mandatory fields have been filled
+  let validateForm = () => {
   };
 
-  let fileTracker = () => {
-    //Tracks how many files
+  //test all inputs
+  useEffect(() => {
+    console.log(
+      'summary', summary,
+      'name', name,
+      'review', review,
+      'email', email,
+      'recommend', recommend,
+      'starRating', starRating,
+      'charRating', charRating,
+      'photoArray', photoArray,
+    );
+  });
+
+  //submits form to server
+  // eslint-disable-next-line camelcase
+  let handleSubmit = (event) => {
+    event.preventDefault;
```

Comment:
not sure if it needs to be event.preventDefault()?

Reply:
Agreed, it should be event.preventDefault();

Reply:
changed. Thank you.